make/modules/jdk.incubator.vector/Lib.gmk

```diff
@@ -74,4 +74,18 @@ ifeq ($(call isTargetOs, linux)+$(call isTargetCpu, aarch64)+$(INCLUDE_COMPILER2
   TARGETS += $(BUILD_LIBSLEEF)
 endif
 
+ifeq ($(call isTargetOs, macosx)+$(call isTargetCpu, aarch64)+$(INCLUDE_COMPILER2), true+true+true)
+  $(eval $(call SetupJdkLibrary, BUILD_LIBSLEEF, \
+      NAME := sleef, \
+      OPTIMIZATION := HIGH, \
+      SRC := libsleef/lib, \
+      EXTRA_SRC := libsleef/generated, \
+      DISABLED_WARNINGS_gcc := unused-function sign-compare tautological-compare ignored-qualifiers, \
```

Comment:
DISABLED\_WARNINGS\_gcc is technically not needed, gcc is not a supported compiler on macOS, at least, not yet...

If you feel that gcc support for macOS is a worthy addition to have to make the lives of future compiler porters, you can leave it in there. It's otherwise up to you whether you want to remove it or not (I personally like the idea of being able to compile for macOS with gcc in the future however)

Reply:
No, we should not have dead code "just in case". If someone were to support gcc on macos (seems implausible if that could ever be possible), then this is the least of their troubles.